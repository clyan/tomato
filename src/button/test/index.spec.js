import { mount } from '@vue/test-utils';
import Button from '../../button';

test('renderbutton', () => {
  const wrapper = mount(Button);
  expect(wrapper).toMatchSnapshot();
});
