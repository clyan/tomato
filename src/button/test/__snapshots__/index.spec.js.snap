// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderbutton 1`] = `
<button class="tomato-button tomato-button--default tomato-button--normal">
  <div class="tomato-button__content"></div>
</button>
`;
